pkgs/development/python-modules/nbclient/default.nix

```diff
@@ -1,27 +1,65 @@
-{ lib, buildPythonPackage, fetchPypi, pythonOlder,
-  async_generator, traitlets, nbformat, nest-asyncio, jupyter-client,
-  pytest, xmltodict, nbconvert, ipywidgets
+{ async_generator
+, buildPythonPackage
 , doCheck ? true
+, fetchFromGitHub
+, ipywidgets
+, jupyter-client
+, lib
+, nbconvert
+, nbformat
+, nest-asyncio
+, pytest-asyncio
+, pytestCheckHook
+, pythonOlder
+, traitlets
+, xmltodict
 }:
 
 buildPythonPackage rec {
   pname = "nbclient";
-  version = "0.5.13";
+  version = "0.6.2";
+  # As of version 0.6.2, there is both a setup.py and pyproject.toml. However,
+  # the pyproject.toml does not appear to be the main entry point.
```

Comment:
```suggestion
```
pyproject.toml can be used for linting tool settings. I don't think we need a comment for that each time.

Reply:
maybe but this is already a source of confusion without the comment as evidenced in other reviews: https://github.com/NixOS/nixpkgs/pull/171563#discussion_r865310873 😜

I was also confused by this when I first read the code!